src/SampleFormat.cpp

```diff
@@ -43,11 +43,10 @@
 #include <string.h>
 
 #include "Prefs.h"
-#include "Dither.h"
 #include "Internat.h"
 
-static DitherType gLowQualityDither = DitherType::none;
-static DitherType gHighQualityDither = DitherType::none;
+DitherType gLowQualityDither = DitherType::none;
+DitherType gHighQualityDither = DitherType::none;
```

Comment:
I'm suggesting DitherType::shaped in place of DitherType::none
A person reading the code who is only incidentally interested will see the right/default setting.
A person who is needing to dig deeper into the code will see this is also the default set by preferences.